components/Form/EditStationForm.jsx

```diff
@@ -99,18 +96,18 @@ export default function EditStationForm({ station }) {
     setSelectedSensors(data)
   }
 
+  if (sensorStatusQuery === 'loading') {
+    return <p>Loading...</p>
+  }
+
+  if (sensorStatusQuery === 'error') {
+    return <p>Error loading sensor data</p>
+  }
+
   return (
     <>
       <PageHeader title={t('title')} className={'inline-flex pl-5'} />
       <form onSubmit={handleSubmit} className="flex flex-col gap-3 max-w-xl">
-        {isError ? (
```

Comment:
Same question as before how do you manage error after submitting the edited station